source/jormungandr/jormungandr/autocomplete/kraken.py

```diff
@@ -77,6 +83,13 @@ def get(self, request, instance):
         if len(resp.places) == 0 and request['search_type'] == 0:
             req.places.search_type = 1
             resp = instance.send_and_receive(req)
+
+        # add bano into the feed publisher if not existent
+        has_bano = next((True for f in resp.feed_publishers if f.id == FEED_PUBLISHER_BANO['id']), None)
```

Comment:
indeed, it's not necessary to do that for kraken, i did it just for the sake of coherence regarding to bragi